storybookjs/storybook
# CI: Introduce `ci:normal` label

Closes https://github.com/storybookjs/storybook/issues/23278



## What I did

I have introduced the `ci:normal` label, which has to be applied to PRs to trigger a normal CircleCI run with a basic set of sandboxes.

## How to test

- apply the labels `ci:daily`, `ci:merged` and `ci:normal` one after another on this pr and see, whether the appropriate circleci job is started (I think this does not work, since the github action file is take from `next` instead of this branch)
- not applying any of the labels will let Danger.js fail, saying, that one of them has to be applied

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
